src/Sepes.Infrastructure/Service/VirtualMachine/VirtualMachineSizeService.cs

```diff
@@ -101,6 +104,7 @@ public async Task UpdateVmSizeCache(CancellationToken cancellationToken = defaul
                     {
                         if (existingSizeItemsForRegion.TryGetValue(curAzureSku.Name, out curVmSizeInDb))
                         {
+                            curVmSizeInDb.Price = _azureCostManagementService.GetSizePrice(curVmSizeInDb.Key).Result;
```

Comment:
I am not quite sure where to put this.. it is in multiple places right now. Maybe you can give me a tip since you know the logic better?